setup.py

```diff
@@ -3,9 +3,9 @@
     name='consensus',
     version='1.0',
     packages=find_packages(),
-    install_requires=['molgenis-py-client>=2.1.0', 'progressbar2==3.39.3', 'termcolor==1.1.0', 'yaspin==0.14.3', 'pandas'],
+    install_requires=['progressbar2==3.39.3', 'termcolor==1.1.0', 'yaspin==0.14.3', 'pandas'],
     author='Mariska Slofstra',
     license='GNU Lesser General Public License 3.0',
     test_suite='nose.collector',
-    tests_require=['nose', 'parameterized']
+    tests_require=['nose', 'parameterized', 'mock']
 )
```

Comment:
It might be interesting to look at possible other metadata to add to the `setup.py` (`description`, `url`, etc). Probably not that important right now, but still worth considering at some point.